bundle/manifests/sailoperator.clusterserviceversion.yaml

```diff
@@ -217,6 +217,12 @@ spec:
         displayName: Helm Values
         path: values
       version: v1alpha1
+    - description: IstioRevisionTag references a Istio or IstioRevision object and
```

Comment:
Why it is possible to reference Istio resource? Shouldn't the tag only reference the IstioRevision? What happens when there are more revisions for given istio? e.g. during the upgrade?

Reply:
Ah, ok. Answer is in the SEP document